src/components/bbox/Bbox.tsx

```diff
@@ -3,40 +3,51 @@ import styled from 'styled-components';
 
 import './bbox.scss';
 
-const bboxBg = 'rgba(255, 255, 255, 0)';
 const bboxBorder = 'grey';
 const bboxBorderHover = 'orangered';
 const bboxRelatedBorder = 'rgba(255,176,0,0.5)';
-const bboxBgSelected = 'rgba(255, 69, 0, 0.5)';
-const bboxRelatedBackground = 'rgba(255,176,0,0.3)';
+const bboxSrtucturedBorder = 'rgba(255,255,255,0)';
```

Comment:
Please change the border color of structure items to orange
![image](https://github.com/veraPDF/verapdf-js-viewer/assets/31737885/6f96f019-ced6-4a56-b405-dc1b9d73b657)